pkg/list/lister.go

```diff
@@ -101,7 +103,8 @@ func (l *lister) GetPackagesWithStatus(
 			}
 		}
 		hasIncludedItems := len(ls) > 0
-		if hasIncludedItems || (!options.OnlyInstalled && !options.OnlyOutdated) {
+		if hasIncludedItems ||
+			(!options.OnlyInstalled && !options.OnlyOutdated && options.Namespace == "" && options.PackageName == "") {
```

Comment:
IMO, `glasskube list quickwit` should still show the "not installed" line for that package if applicable, but with this change the output is "no package found" instead. 

I think checking the namespace makes sense but the "package name" part should be removed.

```suggestion
		if hasIncludedItems ||
			(!options.OnlyInstalled && !options.OnlyOutdated && options.Namespace == "") {
```

Reply:
The "not installed" packages are now also shown if the package name matches. All "not installed" packages are shown if no package name is specified.